mlrun/common/schemas/common.py

```diff
@@ -41,3 +43,44 @@ class ImageBuilder(pydantic.BaseModel):
 
     class Config:
         extra = pydantic.Extra.allow
+
+
+class LabelsModel(pydantic.BaseModel):
```

Comment:
I believe you can do something like
```
Labels = Annotated[AfterValidator(validate_labels)]
```
And then
```
labels: Labels = Query([], alias="label"),
```

Reply:
https://docs.pydantic.dev/latest/concepts/validators/

Reply:
@alonmr `AfterValidator` is not supported in Pydantic v1

Reply:
@alonmr We cannot get the labels as `labels: Labels = Query([], alias='label')` because it is required to be sent in the body and not as a parameter
`AssertionError: Param: labels can only be a request body, using Body()`